teletext-twitter/output.py

```diff
@@ -86,19 +102,34 @@ def write_tweets(twitter_object, mode, count, config, query=None): # grab the la
         tweet_human_time = time.strftime("%d-%b-%Y %H:%S", tweet_time) # reformat time/date output
         tweet_username = charsub(status.user.screen_name)
 
-        with open(filename, "a") as file:
+        with open(filename, "a", newline="") as file:
             post_length = len(tweet_text) + 1 # how long is our next tweet? (including info line)
-            if (line_position + post_length) > 25: # are we about to go over the page?
-                for blankline_position in range(line_position, 25): # how many blank lines do we need?
-                    file.write("OL,{},".format(blankline_position) + (" " * 15) + "\r\n")
+            if (line_position + post_length) > 24: # are we about to go over the page?
                 file.write("FL,0,0,0,0,0,100\r\n")
+                if subpage_enhancements:
+                    write_ehancements(file, subpage_enhancements)
                 subpage += 1 # start a new page
+                subpage_enhancements = []
                 if subpage > 99:
-                    break # reached subpage limit - dump the rest
+                    return # reached subpage limit - dump the rest
```

Comment:
Is there a reason to use return instead of break for this? It was originally just to break out of the loop. Was also tempted to raise a custom exception and read it in __main__.

Reply:
That was my lazy way of being able to finish the page with the FL and any enhancements without getting two copies if it hit the limit. I'm sure there's a less hacky way to do it.